packages/picasso-rich-text-editor/src/LexicalEditor/styles.ts

```diff
@@ -165,7 +165,12 @@ export default (theme: Theme) => {
       },
     },
     code: codeStyles(theme),
-    codeBlock: codeBlockStyles(theme),
+    codeBlock: {
+      '&': codeBlockStyles(theme),
+      '& > span': {
+        fontFamily: 'monospace',
+      },
```

Comment:
Maybe then:
```suggestion
      '& *': {
        fontFamily: 'monospace !important',
      },
```

I think this a valid place to use `!important`. There's no other font that could be technically be applied to code blocks in any other case

Reply:
Checked, works. But, I wouldn't use `!important` here until we are faced with another issue.

Reply:
Updated